public/forgotPass.html

```diff
@@ -12,7 +12,7 @@
     <title>Forget Password Page</title>
     <link rel="stylesheet" href="./css/styles.css" />
     <link rel="stylesheet" href="./css/forgotPass.css">
-    <link rel="stylesheet" href="./css/admin.css">
+    <link rel="stylesheet" href="./css/account.css">
     <script src="/socket.io/socket.io.js"></script>
     <script src="js/chat.js"></script>
```

Comment:
the socket io and chat.js script need to be removed

Reply:
Got it. I was wondering about that. Is that just in the forgotPass.html file?